### PR TITLE
Just updated to latest versions of nginx and docker-gen. 

### PR DESCRIPTION
**Pull request by wfrank2509**
No fundamental changes, only updated versions. Maybe you like to update too?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+Customized version based on jwilder/nginx-proxy! Just updated to recent versions.
+
+
```
Comment (wader) [inline README.md]: Don't include this i guess?

Comment (wfrank2509) [inline README.md]: yes sure ... not that! :)

**Comment by michaellopez** [comment]
@wfrank2509 [0.5.0](https://github.com/jwilder/docker-gen/releases/tag/0.5.0) got released around the time this PR was created :)

**Comment by michaellopez** [comment]
@wfrank2509 Also see #311 for nginx

**Comment by sdurrheimer** [comment]
Current image uses `nginx:1.9.12` and `DOCKER_GEN_VERSION 0.7.0`, this PR can be closed.

**Pull request closed without merging**